odoo/addons/base/tests/test_orm.py

```diff
@@ -38,7 +38,6 @@ def test_access_deleted_records(self):
     def test_access_partial_deletion(self):
         """ Check accessing a record from a recordset where another record has been deleted. """
         Model = self.env['res.country']
-        self.assertTrue(type(Model).display_name.automatic, "test assumption not satisfied")
```

Comment:
The test below depends on the fact that `display_name` is computed and non-stored.